### PR TITLE
[Test] Add support for linux untar tests in nfs

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description
Tests included

1. Perform Linux untar from 1 client and do readir operation from other client (ls -lart)
2. Perform Linux untar from 1 client and do readir operation from other client (du -sh)
3. Perform Linux untar from 1 client and do readir operation from other client (finds)

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/nfs/nfs_verify_readdir_ops.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    clients = ceph_cluster.get_nodes("client")
+
+    port = config.get("port", "2049")
+    version = config.get("nfs_version", "4.0")
+    nfs_node = nfs_nodes[0]
+
+    try:
+        setup_nfs_cluster(clients, nfs_node, port, version)
+
+        # Linux untar on client 1
```
Comment (msainiRedhat) [inline tests/nfs/nfs_verify_readdir_ops.py:74]: Need to parallelise the operations
Eg - TC -1 Linux untars and du's running in parallel until untar is completed
TC - 2 Linux untars and find's running in parallel until untar is completed
TC - 3 Linux untars and ls -laRt running in parallel until untar is completed

**Review by msainiRedhat — CHANGES_REQUESTED** [review]

**Review by msainiRedhat — DISMISSED** [review]

**Comment by pranavprakash20** [comment]
Run log: https://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YX9TEI/

File: tests/nfs/nfs_verify_readdir_ops.py (hunk 1 above)
Comment (pranavprakash20) [reply tests/nfs/nfs_verify_readdir_ops.py:74]: Done

**Review by vamahaja — COMMENTED** [review]
LGTM, @msainiRedhat please review

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by vamahaja — APPROVED** [review]

**Pull request merged**